docs/MockFunctionAPI.md

```diff
@@ -644,3 +644,37 @@ test('direct usage', () => {
   expect(jest.mocked(console.log).mock.calls).toHaveLength(1);
 });
 ```
+
+### `jest.Spied<Source>`
+
+Constructs the type of a spied class or function (i.e. the return type of `jest.spyOn()`).
+
+```ts title="__utils__/setDateNow.ts"
```

Comment:
Now I am happy about the example. The point was to show that typings can be useful in some test util.